src/three/plugins/SUBTREELoader.d.ts

```diff
@@ -0,0 +1,9 @@
+import { LoadingManager } from 'three';
+
+export class SUBTREELoader {
+
+	constructor( manager : LoadingManager, tile, rootTile );
+	load( url : String ) : void ;
+	parse( buffer : ArrayBuffer ) : void ;
+
+}
```

Comment:
We're not exporting the SUBTREELoader so we don't need a d.ts file. I've only been adding these for "public" files to avoid adding more than necessary to maintain.